git/repo.go

```diff
@@ -1,13 +1,18 @@
 package git
 
 import (
+	"bufio"
+	"bytes"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path"
 	"path/filepath"
 	"strings"
+
+	"github.com/G-Node/gin-repo/wire"
```

Comment:
I tried to keep the wire formatting separated so that the `git` module would really only deal with low-level git suff and be encapsulated from all the rest.